docs/cristin.ttl

```diff
@@ -944,12 +951,47 @@ It is a self reference.""" ;
             entityDescription.reference.publicationContext.publisher
         """ .
 
+:TYPE_BOK_RAPPORT_DEL
```

Comment:
PR already exists for this part: https://github.com/BIBSYSDEV/nva-publication-api/pull/514. Remove from here and add comments if necessary to that PR